docs/overview.asciidoc

```diff
@@ -0,0 +1,39 @@
+[[overview]]
+== Overview 
+
+--
+Welcome to APM Server Docs.
+
+NOTE: The documentation pages are still work in progress.
+For more details also check APM Server https://github.com/elastic/apm-server[Github repository].
+
+APM Server is a central component in the Elastic APM system.
+It receives data from APM agents and transforms them into Elasticsearch documents.
+
+To get an overview about the whole Elastic APM system, please also have a look at the documentation for
```

Comment:
I think we should add line breaks after commas as well as periods

Reply:
will change some line breaks.